bukuserver/filters.py

```diff
@@ -124,15 +121,15 @@ def clean(self, value):
 
 
 class BookmarkOrderFilter(BaseFilter):
-    DIR_LIST = [('asc', 'natural'), ('desc', 'reversed')]
+    DIR_LIST = [('asc', _l('natural')), ('desc', _l('reversed'))]
     FIELDS = ['index', 'url', 'title', 'description', 'tags']
 
     def __init__(self, field, *args, **kwargs):
         self.field = field
         super().__init__('order', *args, options=self.DIR_LIST, **kwargs)
 
     def operation(self):
-        return 'by ' + self.field
+        return _l(f'by {self.field}')
```

Comment:
Leaving `_l('by ' + self.field)` would result in pybabel misdetecting a `"by "` translation key :sweat_smile: